api/v1alpha1/managednotification_types.go

```diff
@@ -245,15 +245,6 @@ func (conditions Conditions) GetCondition(t NotificationConditionType) *Notifica
 	return nil
 }
 
-// NewNotificationRecord adds a new notification record status for the given name
```

Comment:
I've doublechecked that this is safe, which it is, but in future I'd recommend in general that we leave this sort of API change to either a separate PR or at least a separate commit, as it's not really related to the intent of the PR/ticket.

Reply:
Ack, I agree that new PR/ticket will be good from tracking perspective. 
Thanks.